i2c/oled_i2c/README.adoc

```diff
@@ -1,36 +1,36 @@
 = Attaching an OLED display via I2C
 
-This example code shows how to interface the Raspberry Pi Pico with an 128x32 OLED display board based on the SSD1306 display driver.
+This example code shows how to interface the Raspberry Pi Pico with an 128x32 OLED display board based on the SSD1306 display driver, datasheet https://cdn-shop.adafruit.com/datasheets/SSD1306.pdf[here].
 
 The code displays a series of tiny raspberries that scroll horizontally, in the process showing you how to initialize the display, write to the entire display, write to only a portion of the display, and configure scrolling.
 
-In contrast to the sensors in the examples library, the SSD1306 is not operated via direct reads/writes to registers. Instead, a list of versatile commands is defined (see datasheet) that allows the user to access all the capabilities of the driver. Ultimately, these commands end up writing to internal registers but the key point is that the I2C write process is now different. After sending a slave address, the data that follows can be either a command, flags to follow up a command or data to be written directly into the display's RAM. A control byte is required for each write after the slave address so that the driver knows what type of data is being sent.
+The SSD1306 is not operated via direct reads/writes to registers. Instead, a list of versatile commands is defined (see datasheet) that allows the user to access all the capabilities of the driver. Ultimately, these commands end up writing to internal registers but the key point is that the I2C write process is now different. After sending a slave address, the data that follows can be either a command, flags to follow up a command or data to be written directly into the display's RAM. A control byte is required for each write after the slave address so that the driver knows what type of data is being sent.
```

Comment:
Perhaps the whole "Ultimately, these commands end up writing to internal registers but the key point is that the I2C write process is now different." sentence is superfluous? People probably only care about the commands they send to the SSD1306 chip, and are less interested about what the chip does behind the scenes? :man_shrugging:

Reply:
Yeah, fair. This was probably me rationalizing the dense language I had read in the datasheet 😅